patches/api/0348-Add-player-health-update-API.patch

```diff
@@ -0,0 +1,44 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: SamB440 <sam@islandearth.net>
+Date: Wed, 17 Nov 2021 12:30:36 +0000
+Subject: [PATCH] Add player health update API
+
+
+diff --git a/src/main/java/org/bukkit/entity/Player.java b/src/main/java/org/bukkit/entity/Player.java
+index 9a6e410206852029f1fea0c4409352d5743dcf64..c5aaebeb2a1ea61b526c3763d6ad687d2746c35f 100644
+--- a/src/main/java/org/bukkit/entity/Player.java
++++ b/src/main/java/org/bukkit/entity/Player.java
+@@ -1558,6 +1558,33 @@ public interface Player extends HumanEntity, Conversable, OfflinePlayer, PluginM
+      */
+     public double getHealthScale();
+ 
++    // Paper start
++    /**
++     * Forcefully sends a health update to the player's client.
```

Comment:
"to the player.", and remove the empty line below this. same for the other method